### PR TITLE
Fix overindented bullet/enumerated lists.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/mpl_toolkits/mplot3d/art3d.py [hunk 1]
```diff
@@ -43,7 +43,9 @@ def get_dir_vector(zdir):
     Parameters
     ----------
     zdir : {'x', 'y', 'z', None, 3-tuple}
+
```
Comment (timhoffm) [inline lib/mpl_toolkits/mplot3d/art3d.py:46]:
```suggestion
```

File: lib/mpl_toolkits/mplot3d/axes3d.py [hunk 2]
```diff
@@ -2724,16 +2725,16 @@ def voxels(self, *args, facecolors=None, edgecolors=None, shade=True,
...
-
-              - A single color value, to color all voxels the same color. This
-                can be either a string, or a 1D rgb/rgba array
-              - ``None``, the default, to use a single color for the faces, and
-                the style default for the edges.
-              - A 3D ndarray of color names, with each item the color for the
-                corresponding voxel. The size must match the voxels.
-              - A 4D ndarray of rgb/rgba data, with the components along the
-                last axis.
+            These parameter can be:
```
Comment (timhoffm) [inline lib/mpl_toolkits/mplot3d/axes3d.py:2728]:
```suggestion
            These parameters can be:
```

**Comment by anntzer** [comment]
yup

**Comment by timhoffm** [comment]
Anybody can merge after CI pass.

**Pull request merged**